app/models/grda_warehouse/hud/project.rb

```diff
@@ -634,26 +617,18 @@ def self.related_item_keys
       [:OrganizationID]
     end
 
-    def self.project_type_override
-      p_t[:computed_project_type]
-      # cl(p_t[:act_as_project_type], p_t[:ProjectType])
-    end
-
-    def compute_project_type
-      act_as_project_type.presence || self.ProjectType
-    end
-
     # Originally wasn't PH, but is overridden to PH
     def project_type_overridden_as_ph?
       @psh_types ||= HudUtility2024.residential_project_type_numbers_by_code[:ph]
       ! @psh_types.include?(self.ProjectType) &&
-        @psh_types.include?(compute_project_type)
+        @psh_types.include?(loaded_items_2024.last&.ProjectType)
```

Comment:
So, with the move to overriding the imported data the difference in value moved from between `ProjectType` and `computed_project_type` to `ProjectType` on the Imported (or post-processed) tables and the equivalent `ProjectType` in the loaded table.  So this now checks the last loaded data set for this record had the same ProjectType.
  
That said, this could definitely use some further thought.  
* The loaded_items for the record should probably be version agnostic
* Instances where the data comes from our own HMIS, there may not be a loaded item, in which case we should short-circuit since there are no overrides there
* loaded_items might be a string instead of an integer, need to double check. 

Thanks for flagging this!
At the moment this is only used during import to determine if a move-in date should be assigned, so maybe the logic should be completely moved to `def self.assign_move_in_date(row)` and pulled out of project.